WebRoot/messages/ZmSMS_fr_CA.properties

```diff
@@ -777,4 +777,4 @@ carrier_USVirginIslands_CentennialWireless.pattern = {0}@cwemail.com
 
 carrier_USVirginIslands_TracFone.label = TracFone
 
-carrier_USVirginIslands_TracFone.pattern = {0}@mmst5.tracfone.com
\ No newline at end of file
+carrier_USVirginIslands_TracFone.pattern = {0}@mmst5.tracfone.com
```

Comment:
It's inconsistent with the other files, and it makes checking for inconsistencies easier.
Functionally I am not sure it matters.